docs/content/en/tasks/monitor/vector/_index.md

```diff
@@ -2,16 +2,16 @@
 title: "Using Vector with k8ssandra-operator"
 linkTitle: "Vector"
 weight: 6
-description: "Configure Vector to scrape metrics"
+description: "Configure Vector"
 ---
 
-Since k8ssandra-operator v1.5.0, it is possible to scrape Cassandra metrics with the Vector agent and customize its configuration to add transformers and sinks.
+k8ssandra-operator provides a Vector instance that can be used to scrape logs and metrics. It's configuration is fully customizable and one can add transformers and sinks to provide the kind of solution user needs.
 
 More information about Vector can be found in the [official documentation](https://vector.dev/docs/).
 
 ## Enabling Vector agent
 
-To enable Vector agent, you need to add a `.spec.cassandra.telemetry` section in the `K8ssandraCluster` manifest:
+Vector agent is enabled by default for Cassandra pods. To enable Vector agent for all the other pods, you need to add a `.spec.cassandra.telemetry` section in the `K8ssandraCluster` manifest:
```

Comment:
```suggestion
Vector agent is enabled by default for Cassandra pods. To enable Vector agent for Stargate and Reaper, you need to add a `.spec.stargate.telemetry` and a `.spec.reaper.telemetry` sections respectively in the `K8ssandraCluster` manifest with `.vector.enabled: true`:
```

The example below should probably not be using `.spec.cassandra`  but rather `.spec.stargate` for example.